Agreements/Internal/Gal_Tal.txt

```diff
@@ -1,3 +1,3 @@
-COPX:*:*:TLA:EGPH:*:Galloway:Talla:*:13000:TLA25
+COPX:*:*:TLA:EGPH:*:Galloway:Talla:*:17000:TLA25
```

Comment:
I would change these slightly so that the 17000 restriction isn't hard coded into the descent profile at TLA:

```suggestion
COPX:GIRVA:*:*:EGPH:*:Galloway:Talla:*:17000:|TLA
COPX:TUNSO:*:*:EGPH:*:Galloway:Talla:*:17000:|TLA
```